standard/standard/clause_7_normative_text.adoc

```diff
@@ -125,28 +125,7 @@ image::diagram/Basic-YPR_SDU.PNG[]
 
 *Requirements Class*
```

Comment:
Remove this line, the labels "Requirements class", "Requirement" etc. are automatically given.

Reply:
Done.